microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -318,6 +327,7 @@ def _kdump_test(self, node: Node, log_path: Path, log: Logger) -> None:
                 shell=True,
                 sudo=True,
                 timeout=10,
+                kill_on_timeout=False
```

Comment:
Use the timeout tool, it can kill the process after timeout.

Reply:
The `execute` function tries to kill the process if the process times-out. In this case the process that gets timed-out is `echo c > /proc/sysrq-trigger`. Trying to kill it makes no sense. So, eventually it goes for a 10-minute unnecessary wait (the timeout of `kill` function is around 10 minute).

With this `kill_on_timeout=False`, `execute` will not try to kill the process after it times-out.

Reply:
LISA has a tool called `Timeout`. You can use it with your commands, so the kill behavior can be controlled in this tool, and it doesn't need to add another argument in the `execute` method.

Reply:
I've updated to use `execute_async` and removed `kill_on_timeout` parameter itself.